tools/check_formatting.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
please add the copyright and license header like in the other shell scripts

Reply:
done